.github/workflows/backlog-automation.yml

```
@@ -1,23 +1,25 @@
-name: Automate project columns
+name: Continuous integration
```

This gets put into the checks section of the pull request, perhaps we could make that a bit terser

```suggestion
name: CI
```

We should probably rename the file to `ci.yml` too